update-release-notes failure on 1.8.3 bug (#2700)
demisto/demisto-sdk: CHANGELOG.md

* Updated **lint** to use graph instead of id_set when running with `--check-dependent-api-module` flag.
* Added the marketplaces field to all schemas.
* Added the flag `--xsoar-only` to the **doc-review** command which enables reviewing documents that belong to XSOAR-supported Packs.

> * Fixed an issue in **update-release-notes** command where an error occurred when executing the command a second time.

Reviewer: ```suggestion
* Fixed an issue in **update-release-notes** command where an error occurred when executing the same command a second time.
```

> * Fixed an issue in **update-release-notes** command where an error occurred during the double execution of the command.

Reviewer: ```suggestion
* Fixed an issue in **update-release-notes** command where an error occurred when executing the command a second time.
```
